examples/opensora_hpcai/opensora/models/vae/autoencoder_kl.py

```diff
@@ -35,6 +36,20 @@ def __init__(
         self.stdnormal = ops.StandardNormal()
         self.split = get_split_op()
 
+        if use_recompute:
+            self.recompute(self.encoder)
+            self.recompute(self.quant_conv)
+            self.recompute(self.post_quant_conv)
+            self.recompute(self.decoder)
+
+    def recompute(self, b):
+        if not b._has_config_recompute:
+            b.recompute()
+        if isinstance(b, nn.CellList):
+            self.recompute(b[-1])
+        else:
+            b.add_flags(output_no_recompute=True)
```

Comment:
Change to this to make `recompute` work in Pynative mode
```suggestion
        elif ms.get_context("mode") == ms.GRAPH_MODE:
            b.add_flags(output_no_recompute=True)
```